common/lib/xmodule/xmodule/modulestore/split_mongo/mongo_connection.py

```diff
@@ -208,6 +210,9 @@ def find_matching_course_indexes(self, branch=None, search_targets=None):
             for key, value in search_targets.iteritems():
                 query['search_targets.{}'.format(key)] = value
 
+        if org_target:
+            query['org'] = org_target
+
```

Comment:
Yeah, `search_targets` is intended for particular content that has been pulled up to the root for querying.